src/screens/points/content/PointsContent.tsx

```diff
@@ -142,6 +145,24 @@ export default function PointsContent() {
     ? `https://www.rainbow.me/points?ref=${points.points.user.referralCode}`
     : undefined;
 
+  const getRankChangeIcon = () => {
+    if (rankChange === undefined || isUnranked) return '';
+
+    if (rankChange === 0) return '􁘶';
+
+    if (rankChange > 0) return '􀑁';
+
+    return '􁘳';
+  };
+
+  const getRankChangeIconColor = () => {
+    if (rankChange !== undefined) {
+      if (rankChange === 0) return colors.yellow;
+      else if (rankChange > 0) return green;
+    }
+
+    return colors.red;
+  };
```

Comment:
```suggestion
  const getRankChangeIconColor = () => {
    if (rankChange === undefined || rankChange < 0) return colors.red;

    if (rankChange === 0) return colors.yellow;

    return colors.green;
  };
```
Nitpick, but I'm a big fan of exiting early on checks so we don't need nested ifs and else clauses.